src/components/donation.tsx

```diff
@@ -65,161 +36,116 @@ export default function Donation() {
     };
 
     return (
-        <Modal isOpen={true} onClose={handleBack} size="2xl">
-            <ModalOverlay />
-            <ModalContent>
-                <ModalHeader>{t('donation.title')}</ModalHeader>
-                <ModalCloseButton />
-
-                <ModalBody paddingBottom={10}>
-                    <Stack spacing={3}>
-                        <Text>{t('donation.content1')}</Text>
-                        <Text>
-                            {t('donation.content2')}
-                            <Link
-                                color="blue.500"
-                                href={`https://${window.location.hostname}/?app=rmp&searchParams=wenxi`}
+        <Modal opened onClose={handleBack} size="xl" title={t('donation.title')}>
+            <Stack>
+                <Text>{t('donation.content1')}</Text>
```

Comment:
I think this is header instead of body ):

Reply:
Not sure if I got this right... Including the screenshots of both envs 

![image](https://github.com/user-attachments/assets/5ec53d99-4abf-45a6-9669-93da70a4fe36)

![image](https://github.com/user-attachments/assets/5e994c50-db54-4bfa-9b57-d5e2fb3b76c9)